CUJExecutor/app/src/androidTest/java/com/example/firstapp/ShellUtility.java

```diff
@@ -582,4 +582,24 @@ public void executeCUJ(String preStr, String postStr, int iterations, boolean re
             logData(allActionStamps);
         }
     }
+
+
+    public void singleCujRun(String preStr, String cujStr, boolean includeMeasured) throws Exception {
```

Comment:
1. method names should generally be verbs
2. what is `includeMeasured`?
3. `preStr` -> `preActionsStr`
4. `cujStr` -> `cujActionsStr`